docker/Dockerfile-tao-release

```diff
@@ -12,7 +12,7 @@ RUN apt-get install -y software-properties-common
 RUN apt-add-repository ppa:ondrej/php
 RUN apt-get update
 RUN apt-get install -y php7.4 php7.4-pdo php7.4-mbstring php7.4-dom php7.4-zip php7.4-gd php7.4-curl npm git jq python3 g++ make curl
-RUN curl -sL https://deb.nodesource.com/setup_12.x | bash
+RUN curl -sL https://deb.nodesource.com/setup_18.x | bash
```

Comment:
We need to release those changes as a new Docker image version on docker hub https://hub.docker.com/r/taotesting/tao-release 

Instructions provided in private message :)